services/brig/src/Brig/User/API/Search.hs

```diff
@@ -104,7 +142,34 @@ search searcherId searchTerm maxResults = do
           else do
             -- For team users, we need to check the visibility flag
             Intra.getTeamSearchVisibility t >>= return . handleTeamVisibility t
-  searchIndex searcherId teamSearchInfo searchTerm maxResults
+  Q.searchIndex searcherId teamSearchInfo searchTerm maxResults
   where
     handleTeamVisibility t Team.SearchVisibilityStandard = Search.TeamAndNonMembers t
     handleTeamVisibility t Team.SearchVisibilityNoNameOutsideTeam = Search.TeamOnly t
+
+browseTeamH ::
```

Comment:
if you want to change the path, you should also change the name of this function to match it.

Reply:
Renamed to teamUserSearch

Reply:
there's more, including module names:

```
$ git grep -Hni browseteam
libs/wire-api/src/Wire/API/User/Search.hs:135:-- | Returned by 'browseTeam' under @/contacts/browse-team/:tid@.
services/brig/brig.cabal:100:      Brig.User.Search.BrowseTeam
services/brig/src/Brig/User/API/Search.hs:31:import qualified Brig.User.Search.BrowseTeam as Q
services/brig/src/Brig/User/API/Search.hs:68:  get "/search/browse-team/:tid" (continue browseTeamH) $
services/brig/src/Brig/User/API/Search.hs:150:browseTeamH ::
services/brig/src/Brig/User/API/Search.hs:161:browseTeamH (_ ::: uid ::: tid ::: mQuery ::: mRoleFilter ::: mSortBy ::: mSortOrder ::: size) = do
services/brig/src/Brig/User/API/Search.hs:162:  json <$> browseTeam uid tid mQuery (undefined mRoleFilter) mSortBy mSortOrder size
services/brig/src/Brig/User/API/Search.hs:164:browseTeam ::
services/brig/src/Brig/User/API/Search.hs:173:browseTeam uid tid mQuery mRoleFilter mSortBy mSortOrder size = do
services/brig/src/Brig/User/API/Search.hs:175:  Q.browseTeam tid mQuery mRoleFilter mSortBy mSortOrder size
services/brig/src/Brig/User/Search/BrowseTeam.hs:21:module Brig.User.Search.BrowseTeam
services/brig/src/Brig/User/Search/BrowseTeam.hs:22:  ( browseTeam,
services/brig/src/Brig/User/Search/BrowseTeam.hs:23:    browseTeamQuery,
services/brig/src/Brig/User/Search/BrowseTeam.hs:37:browseTeam ::
services/brig/src/Brig/User/Search/BrowseTeam.hs:46:browseTeam _tid _mQuery Nothing Nothing Nothing _mSize = do
services/brig/src/Brig/User/Search/BrowseTeam.hs:48:browseTeam _ _ _ _ _ _ = error "not implemented."
services/brig/src/Brig/User/Search/BrowseTeam.hs:54:browseTeamQuery ::
services/brig/src/Brig/User/Search/BrowseTeam.hs:59:browseTeamQuery _team _searchText _mSize = undefined
warning: nested submodules without .gitmodules in the working tree are not supported yet
warning: nested submodules without .gitmodules in the working tree are not supported yet
```